pennylane/devices/preprocess.py

```diff
@@ -177,7 +177,9 @@ def validate_device_wires(
                 new_mp._wires = wires  # pylint:disable=protected-access
                 measurements[m_idx] = new_mp
         if modified:
-            tape = tape.copy(ops=new_ops, measurements=measurements)
+            tape = tape.copy(
```

Comment:
Does any of the other preprocess transforms have the same issue?

Reply:
I am not sure, to be honest. The problem is that this is not a contract for tape transforms that we enforce anywhere. So finding out whether there are *any* issues is about as complex as adding this contract and testing it for all tape transforms we have...